docs/modules/minio.md

```diff
@@ -0,0 +1,48 @@
+# MinIO Containers
+
+Test containers can be used to automatically instantiate and manage [MinIO](https://min.io) containers.
+
+## Usage example
+
+Create a `MinIOContainer` to use it in your tests:
+```java
+import org.testcontainers.containers.MinIOContainer;
+MinIOContainer container = new MinIOContainer("minio/minio:latest");
+```
+
+The [MinIO Java client](https://min.io/docs/minio/linux/developers/java/API.html) can be configured with the container as such:
+```java
+import io.minio.MinioClient;
+MinioClient minioClient = MinioClient
+    .builder()
+    .endpoint(container.getS3URL())
+    .credentials(container.getUserName(), container.getPassword())
+    .build();
+```
+
+If needed the username and password can be overridden with
+```java
+import org.testcontainers.containers.MinIOContainer;
+MinIOContainer container = new MinIOContainer("minio/minio:latest")
+        .withUserName("testuser")
+        .withPassword("testpassword");
+```
```

Comment:
Please take a look at the other modules. Snippets are extracted from source code and not hardcoded.